sdk/ai/ai-projects/review/ai-projects-node.api.md

```diff
@@ -4,402 +4,2548 @@
+    code: string;
+    id: string;
+    message: string;
+}
+
+// @public
+export interface ComputerUsePreviewTool extends Tool {
+    display_height: number;
+    display_width: number;
+    environment: "windows" | "mac" | "linux" | "browser";
+    type: "computer_use_preview";
+}
+
+// @public
+export interface Connection {
+    readonly credentials: BaseCredentialsUnion;
+    readonly id: string;
+    readonly isDefault: boolean;
+    readonly metadata: Record<string, string>;
+    readonly name: string;
+    readonly target: string;
+    readonly type: ConnectionType;
+}
+
+// @public
+export interface ConnectionsGetOptionalParams extends OperationOptions {
+    clientRequestId?: string;
+}
+
+// @public
+export interface ConnectionsGetWithCredentialsOptionalParams extends OperationOptions {
+    clientRequestId?: string;
+}
+
+// @public
+export interface ConnectionsListOptionalParams extends OperationOptions {
+    clientRequestId?: string;
+    connectionType?: ConnectionType;
+    defaultConnection?: boolean;
+}
+
+// @public
+export interface ConnectionsOperations {
+    get: (name: string, includeCredentials?: boolean, options?: ConnectionsGetOptionalParams) => Promise<Connection>;
+    getDefault: (connectionType: ConnectionType, includeCredentials?: boolean) => Promise<Connection>;
+    getWithCredentials: (name: string, options?: ConnectionsGetWithCredentialsOptionalParams) => Promise<Connection>;
+    list: (options?: ConnectionsListOptionalParams) => PagedAsyncIterableIterator<Connection>;
+}
+
+// @public
+export type ConnectionType = "AzureOpenAI" | "AzureBlob" | "AzureStorageAccount" | "CognitiveSearch" | "CosmosDB" | "ApiKey" | "AppConfig" | "AppInsights" | "CustomKeys" | "RemoteTool";
+
+// @public
+export interface ContainerAppAgentDefinition extends AgentDefinition {
+    container_app_resource_id: string;
+    container_protocol_versions: ProtocolVersionRecord[];
+    ingress_subdomain_suffix: string;
+    // (undocumented)
+    kind: "container_app";
+}
+
+// @public
+export type ContinuablePage<TElement, TPage = TElement[]> = TPage & {
+    continuationToken?: string;
+};
+
+// @public
+export interface ContinuousEvaluationRuleAction extends EvaluationRuleAction {
+    evalId: string;
+    maxHourlyRuns?: number;
+    // (undocumented)
+    type: "continuousEvaluation";
+}
+
+// @public
+export interface Coordinate {
+    x: number;
+    y: number;
+}
+
+// @public
+export interface CosmosDBIndex extends Index {
+    connectionName: string;
+    containerName: string;
+    databaseName: string;
+    embeddingConfiguration: EmbeddingConfiguration;
+    fieldMapping: FieldMapping;
+    type: "CosmosDBNoSqlVectorStore";
+}
+
+// Warning: (ae-forgotten-export) The symbol "CreateAgentFromDefinitionConfig" needs to be exported by the entry point index.d.ts
+// Warning: (ae-forgotten-export) The symbol "CreateAgentFromManifestConfig" needs to be exported by the entry point index.d.ts
```

Comment:
Could we make sure all public definitions are exported properly and no such warnings exist in the review doc?